index.html

```diff
@@ -226,76 +226,39 @@ <h1>The Parameter Registry</h1>
 
   <section>
     <h1>The Property Registry</h1>
-
     <p>
-This table summarizes the globally registered properties that can be provided
-as properties in a DID Document.
+      The following section defines the properties in a DID document.
     </p>
-
-    <table class="simple">
-      <thead>
-        <tr>
-          <th>Property</th>
-          <th>Description</th>
-        </tr>
-      </thead>
-
-      <tbody>
-        <tr>
-          <td>
-TBD
-          </td>
-          <td>
-            <div>DESCRIPTION_GOES_HERE</div>
-            <div><a href="">SPECIFICATION_GOES_HERE</a></div>
-            <pre>
-"@context": [
-  "https://www.w3.org/ns/did/v1",
-  "https://CONTEXT_GOES_HERE/v1"
-]
-            </pre>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </section>
-
-
     <section>
-      <h1>The Value Registry</h1>
-
-      <p>
-  This table summarizes the globally registered values that can be used as
-  property values in a DID Document.
-      </p>
-
-      <table class="simple">
-        <thead>
-          <tr>
-            <th>Value</th>
-            <th>Description</th>
-          </tr>
-        </thead>
-
-        <tbody>
-          <tr>
-            <td>
-  TBD
-            </td>
-            <td>
-              <div>DESCRIPTION_GOES_HERE</div>
-              <div><a href="">SPECIFICATION_GOES_HERE</a></div>
-              <pre>
-  "@context": [
-    "https://www.w3.org/ns/did/v1",
-    "https://CONTEXT_GOES_HERE/v1"
-  ]
-              </pre>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <h2><dfn data-lt="publicKey" data-dfn-type="dfn" id="publicKey">publicKey</dfn></h2>
+      <p>An array of public keys. A public key is a verification method. Public keys are used for digital signatures, encryption and other cryptographic operations, which in turn are the basis for purposes such as authentication.</p>
+      <pre class="example">
+        {
+          "id": "did:example:123",
+          "publicKey": [
+            {
+              "id": "did:example:123#ZC2jXTO6t4R501bfCXv3RxarZyUbdP2w_psLwMuY6ec",
+              "type": "Ed25519VerificationKey2018",
+              "controller": "did:example:123",
+              "publicKeyJwk": {
+                "crv": "Ed25519",
+                "x": "o3v3B4wU7LPewgr3iQDSc9LGzJKdS0R_698dUXtBXX8",
+                "kty": "OKP",
+                "kid": "ZC2jXTO6t4R501bfCXv3RxarZyUbdP2w_psLwMuY6ec"
+              }
+            }
+          ]
+        }
+      </pre>
+      <h4>Machine Readable Definitions</h4>
+      <ul>
+        <li><a href="https://w3c.github.io/did-core-registry/schemas/publicKey.json">JSON Schema</a></li>
+        <li><a href="https://w3c.github.io/did-core-registry/#publicKey">JSON-LD</a></li>
```

Comment:
This doesn't tell them which `@context` to use... need to fix that... we should probably point to `https://www.w3.org/ns/did/v1` instead. The other issue is that the spec is the normative definition of publicKey, not the registry. We should point from here to the appropriate section in the spec (which is what these sorts of registries do at IETF).

Reply:
I was not sure exactly how to do this... I'd like the registry to link directly to the line in the JSON-LD context that defines the machine readable variant, if possible, but I can also see the point of just linked to the whole context (https://www.w3.org/ns/did/v1), and letting the person search for the term...

Reply:
The easiest thing to do right now is the latter... we could go fancy and enable an HTML rendering of the context file that has anchors for every line and then use that mechanism to point to specific lines in a .jsonld file. Just a simple matter of slapping some code together for that (that no one has time to do right now). We've wanted to do that for years... no one has stepped up to write that code.

Reply:
Let me take a stab and a programmatic approach, I'd rather just generate this stuff.

Reply:
programatic approach is possible, but its a little brittle. I think its best to hand craft sadly....

Reply:
Can I resolve this?